src/libs/PressEvents.js

```diff
@@ -0,0 +1,16 @@
+import lodashGet from 'lodash.get';
+
+/**
+ * Is this press event a right click on a mouse?
+ *
+ * @param {Object} pressEvent
+ * @returns {Boolean}
+ */
+function isRightClick(pressEvent) {
+    return lodashGet(pressEvent, ['nativeEvent', 'which'], 0) === 3;
```

Comment:
Okay, I see what you mean that this should be handled within `PressableWithSecondaryInteractionHandler`.  I think we basically we we want `onSecondaryInteraction` to trigger:

1) On right-click
2) On long press (but not when that long press is a left-click of a mouse, because that's weird).

Reply:
This thread's not relevant anymore